package.json

```diff
@@ -77,25 +72,24 @@
     "gulp-sourcemaps": "1.6.0",
     "gulp-uglify": "2.0.1",
     "gulp-util": "3.0.8",
-    "jasmine-check": "^0.1.2",
-    "jest": "^17.0.3",
-    "magic-string": "0.10.2",
+    "jasmine-check": "0.1.5",
+    "jest": "19.0.2",
     "marked": "0.3.6",
-    "microtime": "^2.0.0",
```

Comment:
This one is actually an optionalDep of benchmark, so I'm adding this back in a follow-up commit